src/lwc/bdiFieldMappings/bdiFieldMappings.js

```diff
@@ -134,6 +137,10 @@ export default class bdiFieldMappings extends LightningElement {
                     await getObjectFieldDescribes({objectName: this._dataImportApiName});
             }
 
+            // Get mapped data import field describes
+            this.mappedDiFieldDescribes =
```

Comment:
"mappedDiFieldDescribes" makes me think this will contain describe information, but in reality this collection is field api names right? Should we update the name of the variable?